tools/e2etesting/pipeline_orchestrated.yml

```diff
@@ -1,8 +1,27 @@
 name: $(Date:yyyyMMdd)$(Rev:rr)
-
 trigger: none
 pr: none
 
+schedules:
```

Comment:
Let's add scheduling to `pipeline_standalone.yml` as well so that we have nightly runs of both orchestrated and standalone pipeline.